frontend/views/accessions/index.html.erb

```diff
@@ -7,7 +7,7 @@
 
   # Add standard columns minus actions
   model = "accession"
-  add_multiselect_column if can_delete_search_results?(model) && !(request.path =~ /\/(advanced_)*search/)
+  add_multiselect_column if can_delete_search_results?(model)
```

Comment:
I went ahead and removed this extra condition (which I initially copied from `add_columns`, which is used all over the place), since we know the `path` for this template will be `/accessions/`.